nominatim/tokenizer/sanitizers/tag_japanese.py

```diff
@@ -0,0 +1,150 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This file is part of Nominatim. (https://nominatim.org)
+#
+# Copyright (C) 2022 by the Nominatim developer community.
+# For a full list of authors see the git log.
+"""
+This sanitizer maps OSM data to Japanese block addresses.
+It replaces blocknumber and housenumber with housenumber,
+and quarter and neighbourhood with place.
+"""
+
+
+from typing import Callable
+from typing import List
+
+from nominatim.tokenizer.sanitizers.base import ProcessInfo
+from nominatim.tokenizer.sanitizers.config import SanitizerConfig
+from nominatim.data.place_name import PlaceName
+
+def create(_: SanitizerConfig) -> Callable[[ProcessInfo], None]:
+#def create(config: SanitizerConfig) -> Callable[[ProcessInfo],None]:
+    """Set up the sanitizer
+    """
+    return tag_japanese
+    #return tag_japanese(config)
+
+def convert_kanji_sequence_to_number(sequence: str) -> str:
+    """Converts Kanji numbers to Arabic numbers
+    """
+    kanji_map = {
```

Comment:
Looks like a constant which could be reused?

Reply:
[nominatim/tokenizer/sanitizers/kanji_utils.py](https://github.com/osm-search/Nominatim/pull/3122/files#diff-7781f8b352789295d8816ef6558c5b4059dbcc97be9b36196fa08923df4023c7)
I split it into new files.

Reply:
I don' t think this is what was meant here. Rather move ' kanji_map'  outside the function as a module global variable. Then it is initialised only once. Please note, that the variable name should be then in uppercase letters for being a constant.

Reply:
I'm sorry. I was mistaken.
I moved and capitalized.